prime-router/build.gradle.kts

```diff
@@ -819,9 +819,9 @@ dependencies {
         exclude(group = "com.azure", module = "azure-core")
         exclude(group = "com.azure", module = "azure-core-http-netty")
     }
-    implementation("org.apache.logging.log4j:log4j-api:[2.17.1,)")
```

Comment:
A new beta version became available [12/22](https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-api) and the way this dependency is specified means that we picked it up